src/raftstore/store/worker/pd.rs

```diff
@@ -258,6 +258,10 @@ impl<T: PdClient> Runner<T> {
                       pd_region);
                 PD_VALIDATE_PEER_COUNTER_VEC.with_label_values(&["peer valid"]).inc();
             }
+            Ok(None) => {
+                // split region has not yet report to pd.
+                // TODO: handle merge
```

Comment:
Because None is returned, the region can be removed for either late split heartbeat or merged.